packages/chatbot-server-mongodb-public/.env.example

```diff
@@ -11,3 +11,6 @@ MONGODB_DATABASE_NAME=<docs-chatbot-ENV>
 VECTOR_SEARCH_INDEX_NAME=<index_name>
 NODE_ENV=development
 ALLOWED_ORIGINS=http://example.com,http://localhost:5173
+RADIANT_BASE_URL="<some radiant uri>"
+RADIANT_API_KEY="<some radiant API key>"
```

Comment:
I get the `RADIANT` naming for differentiating in the tests but longer term the idea would be that you specify the radiant URL/API key for `OPENAI_ENDPOINT` and `OPENAI_API_KEY` right?

Reply:
> I get the RADIANT naming for differentiating in the tests but longer term the idea would be that you specify the radiant URL/API key for OPENAI_ENDPOINT and OPENAI_API_KEY right?

i'm not sure if it'd matter necessarily. why use the same var?